lib/nerves_network/dhclient.ex

```diff
@@ -140,10 +140,10 @@ defmodule Nerves.Network.Dhclient do
   end
 
   #  Nerves.Network.Dhclient.handle_info({#Port<0.6423>, {:exit_status, 0}}, %{ifname: "eth1", port: #Port<0.6423>})
-  def handle_info({pid, {:exit_status, exit_status}}, state) do
-    Logger.debug fn -> "#{__MODULE__}: handle_info pid = #{inspect pid}: exit_status = #{inspect exit_status}, state = #{inspect state}" end
-    "#{__MODULE__} Exit status: #{inspect exit_status} pid: #{inspect pid}"
-    |> handle_dhclient(state)
```

Comment:
are we certain that removal of the handle_state here is a good idea? - It has been a long time ago since I entered this turf so I do not remember hence am asking